strawberry/extensions/base_extension.py

```diff
@@ -10,6 +11,13 @@
     from strawberry.types import ExecutionContext
 
 
+class LifecycleStep(Enum):
```

Comment:
I think it would fit better in https://github.com/strawberry-graphql/strawberry/blob/51eb183a317722480bc08cfc1032bfdad048e253/strawberry/extensions/context.py#L201

And the name might be `ExtensionLifecycle` / `OperationLifeCycle`
Also I would add a comment there for it's purpose.

@patrick91 What do you think?

Reply:
I think `LifecycleStep` should be ok, since it is namespaced inside the extensions module already 😊